Automattic/jetpack
# CI: Try collect js coverage

#### Changes proposed in this Pull Request:

* Update code coverage collection to include packages and JS files

#### Jetpack product discussion


n/a
#### Does this pull request change what data or activity we track or use?


n/a
#### Testing instructions:






* Check the "Checks" sections. Is code coverage has changed?
* Look through Code Climate report and see if all/most js and PHP files are included: https://codeclimate.com/github/Automattic/jetpack/compare/6d2a95c6da73ad01e1e11b9b13fcd54c9a0e52af...b48848dd0c605e1cd229f8d1c83a6d4437f17325#ratings

#### Proposed changelog entry for your changes:

* n/a
